admin/app/views/workarea/admin/shipping_skus/show.html.haml

```diff
@@ -0,0 +1,16 @@
+- @page_title = @sku.name
+
+.view
+  .view__header
+    .grid
+      .grid__cell.grid__cell--25
+        / left side element needed for spacing
```

Comment:
Here too

Reply:
ah perfect, thanks! I think I had `.grid--auto` or something and that was messing everything up.